test/common/runtime/runtime_impl_test.cc

```diff
@@ -795,7 +795,7 @@ class DiskLayerTest : public testing::Test {
 protected:
   DiskLayerTest() : api_(Api::createApiForTest()) {}
 
-  static void SetUpTestSuite() {
+  static void SetUpTestSuite() { // NOLINT(readability-identifier-naming)
```

Comment:
What does this do compared to SetUp? Wondering if we can use this less.

Reply:
It's like `SetUp` but runs once per the whole test suite. That is once before the first test with `TearDownTestSuite` called after the last test.

I don't think it would be correct to replace it with `SetUp` and in the best case it would just be inefficient.